runtime/src/serde_snapshot.rs

```diff
@@ -244,35 +237,29 @@ where
                 if std::fs::rename(append_vec_abs_path.clone(), target).is_err() {
                     let mut copy_options = CopyOptions::new();
                     copy_options.overwrite = true;
-                    let e = fs_extra::move_items(
+                    if let Err(e) = fs_extra::move_items(
                         &vec![&append_vec_abs_path],
                         &local_dir,
                         &copy_options,
-                    )
-                    .map_err(|e| {
-                        format!(
-                            "unable to move {:?} to {:?}: {}",
+                    ) {
+                        warn!(
+                            "context_accounts_db_from_fields error: unable to move {:?} to {:?}: {:?}",
                             append_vec_abs_path, local_dir, e
-                        )
-                    })
-                    .map_err(accountsdb_to_io_error);
-                    if e.is_err() {
-                        info!("{:?}", e);
-                        continue;
```

Comment:
This ill-taste `continue` was a work-around introduced at: https://github.com/solana-labs/solana/pull/7281/files#diff-2099c5256db4eb5975c8834af38f6456R503

With good confidence, we no longer need this. This isn't no longer warning. It's safe to cause error and exit now as this is really bad condition now. Could you remove `continue` and adjust the new code so that the work-around in that dated pr is effectively reverted?

Reply:
(For the record: this workaround got deprecated at https://github.com/solana-labs/solana/pull/8339)